test/integration/crcsuite/prepare.go

```diff
@@ -120,7 +120,7 @@ func CopyFilesToTestDir() error {
 
 func ParseFlags() {
 
-	flag.StringVar(&bundleURL, "bundle-location", "embedded", "Path to the bundle to be used in tests")
+	flag.StringVar(&bundleLocation, "bundle-location", "embedded", "Path to the bundle to be used in tests")
```

Comment:
If we change [L93 in `crcsuite.go`](https://github.com/code-ready/crc/pull/1826/files#diff-e4fa8ddc3d7d9a4a1c20d74b5492daac1c3e3051e5e3c2d33ee189fb9b60f80bR93), then the default value here should be changed from `"embedded"` to `""` as well.

Reply:
@jsliacan Good catch 👍🏼